### PR TITLE
fix error if not set user timezone

### PR DESCRIPTION
**Pull request by samsagaz**
Fix Error if not set user timezone
Fix Lang Typo

### CODE REVIEW

File: l10n_ar_wsfe/wsfetools/wsfe_easywsy.py [hunk 1]
```diff
@@ -56,11 +56,9 @@ def parse_invoice(self, invoice, number=False, invoices=False):
         if not number:
             number = invoice.split_number()[1]
 
-        date_invoice = datetime.strptime(invoice.date_invoice, '%Y-%m-%d')
+        date_invoice = invoice.date_invoice
         formatted_date_invoice = date_invoice.strftime('%Y%m%d')
```
Comment (gabrielo77) [inline l10n_ar_wsfe/wsfetools/wsfe_easywsy.py:59]: `date_invoice` here isn't a string? I think you need to use `fields.Date.from_string()` to cast that string to a datetime object

File: l10n_ar_wsfe/wsfetools/wsfe_easywsy.py [hunk 2]
```diff
@@ -56,11 +56,9 @@ def parse_invoice(self, invoice, number=False, invoices=False):
...
-        date_due = invoice.date_due and datetime.strptime(
-            invoice.date_due, '%Y-%m-%d').strftime('%Y%m%d') or \
-            formatted_date_invoice
+        date_due = invoice.date_due and invoice.date_due.strftime('%Y%m%d') or formatted_date_invoice
```
Comment (gabrielo77) [inline l10n_ar_wsfe/wsfetools/wsfe_easywsy.py:60]: same as above, but for `date_due`

File: l10n_ar_wsfe/wsfetools/wsfe_easywsy.py [hunk 3]
```diff
@@ -620,24 +616,22 @@ def validate_service_payment_date(val, invoice, Concepto):
...
                 _('WSFE Error!\n') +
                 _('There is another Invoice with a most recent date [%s] ' +
                   'for the same Point of Sale and Denomination.') %
                 last_invoiced_date)
         today = fields.Date.context_today(invoice)
-        today_dt = datetime.strptime(today, DATE_FORMAT)
-        offset = today_dt - val_dt
+        offset = today - val_date.date()
         if Concepto in [2, 3]:
             if abs(offset.days) > 10:
```
Comment (gabrielo77) [inline l10n_ar_wsfe/wsfetools/wsfe_easywsy.py]: it would be nice to have the 10 in a configuration so we don't hardcode it

**Review by gabrielo77 — CHANGES_REQUESTED** [review]

File: l10n_ar_wsfe/wsfetools/wsfe_easywsy.py (hunk 1 above)
Comment (samsagaz) [reply l10n_ar_wsfe/wsfetools/wsfe_easywsy.py:59]: invoice.date_invoice is a datetime.date()

File: l10n_ar_wsfe/wsfetools/wsfe_easywsy.py (hunk 2 above)
Comment (samsagaz) [reply l10n_ar_wsfe/wsfetools/wsfe_easywsy.py:60]: Same as above, already have the obj

File: l10n_ar_wsfe/wsfetools/wsfe_easywsy.py (hunk 3 above)
Comment (samsagaz) [reply l10n_ar_wsfe/wsfetools/wsfe_easywsy.py]: done

File: l10n_ar_wsfe/wsfetools/wsfe_easywsy.py (hunk 2 above)
Comment (gabrielo77) [reply l10n_ar_wsfe/wsfetools/wsfe_easywsy.py:60]: yeah, my bad, it's changed in the latest Odoo version. Thank you!

**Review by gabrielo77 — APPROVED** [review]

**Pull request merged**